specs/spec-1-kudos.md

```diff
@@ -1,107 +1,315 @@
 ---
-#: 1
+Spec ID: 1
 Status: Review
-Category: NEAR Social
-Created: 2023-04-05
-Replaces: --
-Requires: --
+Category: NEAR Social Widget
+Created: 2023-04-21
+Replaces: n/a
+Requires: n/a
 ---
 
-# RFP 2: Kudos
+# Spec 1: Kudos Widget
 
-# Summary
+## Overview
 
-Kudo is a widget to build a reputation using SBT tokens and a network effect. Kudo is a form of a recommendation to share a gratitude or achievement shared between humans.
+<!-- Objective. In a few sentences, describe the key system objectives. Tweet length executive summary. -->
```

Comment:
let's remove comments from this document.

Reply:
I prefer to keep comments in place since they provide direction to maintainers of the living document.

Reply:
* NEP templates requires "Summary": https://github.com/near/NEPs/blob/master/nep-0000-template.md
* W3C is also using Summary
* Latest EIPs are using "Abstract": https://github.com/ethereum/EIPs/blob/master/EIPS/eip-6900.md
* Algorand is using "Abstract": https://github.com/algorandfoundation/ARCs/blob/main/ARC-template.md

TBH, I don't recall any design process which uses "overview"

Reply:
This is considered a bad practice. These comments are in template, and add a noise here (harder to read the source). Other design process also request to remove template comments.

Reply:
A subsequent editor might not go back to the template to see the comments. It's easy enough to hide comments by switching out of the raw/source view.